src/Whip_WPDismissOption.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * Represents the WordPress option for saving the dismissed messages.
+ */
+class Whip_WPDismissOption implements Whip_DismissStorage {
+
+	/**
+	 * Saves the value to the options.
+	 *
+	 * @param string $dismissedVersion The value to save.
+	 */
+	public function set( $dismissedVersion ) {
+		update_option( 'whip_dismissed', $dismissedVersion );
```

Comment:
We should return a boolean: the result of `update_option`
Also reflect in return documentation.